common/src/main/scala/org/apache/celeborn/common/metrics/source/Role.scala

```diff
@@ -20,4 +20,5 @@ package org.apache.celeborn.common.metrics.source
 object Role {
   val MASTER = "master"
   val WORKER = "worker"
+  val RPC = "rpc"
```

Comment:
I believe that the role names should represent a single process. Therefore, it would be more appropriate to use MASTER, WORKER, and CLIENT instead of RPC, WDYT?

Reply:
Good suggestion, I have transferred the Role to Rpc Env Config.